AttributeError: module 'numpy' has no attribute 'math'

All non-`3.8-numba` environments failing with:

```
FAILED esda/tests/test_shape.py::test_reflexive_angle_ratio - AttributeError: module 'numpy' has no attribute 'math'
```

* Example [here](https://github.com/pysal/esda/actions/runs/9847620870/job/27187987821#step:7:627).
* All environments have the same `numpy` version from what I can tell.

 
`numba` environment versions ([here](https://github.com/pysal/esda/actions/runs/9847620870/job/27187987821#step:5:186)):
```
  numba                          0.60.0        py311h4bc866e_0      conda-forge
  numpy                          2.0.0         py311h1461c94_0      conda-forge
```

non-`numba` environment versions ([here](https://github.com/pysal/esda/actions/runs/9847620870/job/27187987262#step:5:238)):
```
  numpy                          2.0.0         py311h1461c94_0      conda-forge
```

--- a/.gitignore
+++ b/.gitignore
@@ -10,7 +10,7 @@ dist
 .bash_history
 .cache/
 .conda/
-.coverage
+*.coverage*
 .coverage.snuc.30509.410533
 .eggs/
 .ipython/

--- a/esda/shape.py
+++ b/esda/shape.py
@@ -123,7 +123,7 @@ def _get_angles(points, n_coords_per_geom):
         a = left - center
         b = right - center
         # compute the angle between the segments
-        angle = numpy.math.atan2(a[0] * b[1] - a[1] * b[0], numpy.dot(a, b))
+        angle = numpy.atan2(a[0] * b[1] - a[1] * b[0], numpy.dot(a, b))
         result.append(angle)
         on_coord += 1
     return result

